docs/duration.md

```diff
@@ -1,17 +1,120 @@
+| **PT2H30M**          | Two and a half hours |
+| **P1M**              | One month |
+| **PT1M**             | One minute |
+| **PT0.0021S**        | 2.1 milliseconds (two milliseconds and 100 microseconds) |
+| **PT0S**             | Zero |
+| **P0D**              | Zero |
 
 ## Constructor
 
 ### **new Temporal.Duration**(_years_?: number, _months_?: number, _days_?: number, _hours_?: number, _minutes_?: number, _seconds_?: number, _milliseconds_?: number, _microseconds_?: number, _nanoseconds_?: number) : Temporal.Duration
 
-Creates a new `Duration` object that represents a duration of time.
+**Parameters:**
+- `years` (optional number): A number of years.
+- `months` (optional number): A number of months.
+- `days` (optional number): A number of days.
+- `hours` (optional number): A number of hours.
+- `minutes` (optional number): A number of minutes.
+- `seconds` (optional number): A number of seconds.
+- `milliseconds` (optional number): A number of milliseconds.
+- `microseconds` (optional number): A number of microseconds.
+- `nanoseconds` (optional number): A number of nanoseconds.
+
+**Returns:** a new `Temporal.Duration` object.
+
+All of the arguments are optional.
+Any missing or `undefined` numerical arguments are taken to be zero, and all non-integer numerical arguments are rounded down to the nearest integer.
+Negative numbers are not allowed.
+
+Use this constructor directly if you have the correct parameters already as numerical values, but otherwise `Temporal.Duration.from()`, which accepts more kinds of input and allows disambiguation behaviour, is probably more convenient.
+
+Usage examples:
+```javascript
+new Temporal.Duration(1, 2, 3, 4, 5, 6, 987, 654, 321)  // => P1Y2M3DT4H5M6.987654321S
+new Temporal.Duration(0, 0, 40)  // => P40D
+new Temporal.Duration(undefined, undefined, 40)  // => P40D
+new Temporal.Duration()  // => PT0S
+```
 
 ## Static methods
 
 ### Temporal.Duration.**from**(_thing_: string | object, _options_?: object) : Temporal.Duration
 
+**Parameters:**
+- `thing` (string or object): A `Duration`-like object or a string from which to create a `Temporal.Duration`.
+- `options` (optional object): An object with properties representing options for constructing the duration.
+  The following options are recognized:
+  - `disambiguation` (optional string): How to disambiguate if any of the other arguments are out of range.
+    Allowed values are `constrain`, `balance`, and `reject`.
+    The default is `constrain`.
+
+**Returns:** a new `Temporal.Duration` object.
+
+This static method creates a new duration from another value.
+The value can be a string in the ISO 8601 format described above.
+Or, if the value is an object, it can either be another `Temporal.Duration` object, or a plain object with properties `years`, `months`, etc.
+
+This function is often more convenient to use than `new Temporal.Duration()` because you can specify only the properties you need.
```

Comment:
Not entirely true, given that the arguments are optional. Readability might be a more obvious advantage.